PostProcessing/master.py

```diff
@@ -24,7 +26,7 @@
 
 def HL_fitting_function(infile, outfilename, func_name="MultiGauss", 
                        num_funcs=2, lenscale=(400,40), plot=None, outfig='./figures', 
-                       nproc=4, min_num_obs=2, max_iter=100):
+                       nproc=4, min_num_obs=2, max_iter=100, f_test=True):
```

Comment:
James, I think setting this to True as default will impact the test results. I think that's fine as long as you update the KGO adding the new output with additional variables. Actually, I think you should include a test with f_test=True and leave the other tests as they are (f_test=False), so we can make sure that we cover everything.

Reply:
Happy to make this change.  However, I didn't know there was any KGO anywhere - I can't see them in the branch. Can you point me to where these are?

The KGOs will need updating anyway due to the name change of chi_sq to RSS.  I also store the RSS_vs_mean value in the file.  This is to facilitate doing a test a-priory even if one isn't done at run-time.

Reply:
Sorry, we don't have KGOs. You're right and I should probably include them at some point.  What I do is to generate them with run_tests before changing the code, so I have a reference. In the meantime, you can set the existing HL fitting tests to have f_test = False and add another test with f_test = True, so at least we have different outputs being generated as result of the tests.

Reply:
I'll add a KGO directory with the test results.

Reply:
Thanks! :)

Reply:
Pushed my code changes back now. You'll see the new KGO directory.  I've also added a new test to test_HL_fitting.py where I delibratly degrade the top right corner of the grid by adding random noise to the correlations and then do the fit.  The purpose of this test is to see if the p_value from my F-test picks this up - which it does so all is good.

I also have seperate tests with the F test both on and off.